package.json

```diff
@@ -32,25 +32,25 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@razee/kubernetes-util": "^1.0.0",
-    "axios": "^0.27.2",
+    "@razee/request-util": "^0.0.4",
```

Comment:
`request-util: 0.0.4` does not contain the `aws` support, so this will need to specify at least `^0.0.5`.... but 0.0.5 with the aws support has not been released yet, so this PR cannot be merged until that is done and confirmed available.  You should put 'DNM' (do not merge' in this PR title and convert it to `draft` until ready to merge.